src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,11 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import React from 'react';
```

Comment:
You are importing React but it's not necessary in this case, since you don't use any React object in this component. Remove this line to make your code cleaner.